tests/KeysTest.elm

```diff
@@ -141,4 +141,20 @@ esc action =
                 , ( "repeat", bool False )
                 ]
     in
-        Event.custom action props
+    Event.custom action props
+
+
+tab : String -> ( String, Value )
+tab action =
+    let
+        props =
+            Encode.object
+                [ ( "keyCode", int Keys.tab )
+                , ( "shiftKey", bool False )
+                , ( "metaKey", bool False )
+                , ( "ctrlKey", bool False )
+                , ( "altKey", bool False )
+                , ( "repeat", bool False )
+                ]
+    in
+    Event.custom action props
```

Comment:
It looks like this is not actually getting called in the tests. The other similarly named helper functions are used in the `eventTests` function to prove various behaviors of the event bindings. Probably better to remove this for now until a use for it arises.

Reply:
Ahh yes, I'll add a usage to the `eventTests` if you prefer that?  At least it would have minimal test coverage?

Reply:
NM, I'll just remove.  The addtion is so simple, there is not much value there to just test an additional `Int` keycode